pkg/cmd/admin/prune/images.go

```diff
@@ -86,6 +88,7 @@ func NewCmdPruneImages(f *clientcmd.Factory, parentName, name string, out io.Wri
 	cmd.Flags().BoolVar(&opts.Confirm, "confirm", opts.Confirm, "Specify that image pruning should proceed. Defaults to false, displaying what would be deleted but not actually deleting anything.")
 	cmd.Flags().DurationVar(&opts.KeepYoungerThan, "keep-younger-than", opts.KeepYoungerThan, "Specify the minimum age of an image for it to be considered a candidate for pruning.")
 	cmd.Flags().IntVar(&opts.KeepTagRevisions, "keep-tag-revisions", opts.KeepTagRevisions, "Specify the number of image revisions for a tag in an image stream that will be preserved.")
+	cmd.Flags().BoolVar(&opts.KeepOverSizeLimitImages, "keep-over-size-limit-images", opts.KeepOverSizeLimitImages, "Specify if images which are exceeding limits (see 'openshift.io/Image') should not be considered for pruning.")
```

Comment:
The same remark as below. It's inaccurate, they will be considered even if over-sized.